Sources/CashUI/UI/OtherUI/MapViewController.swift

```diff
@@ -88,7 +88,7 @@ class MapViewController: UIViewController {
         mapATMs.isZoomEnabled = true
 
         if #available(iOS 13.0, *) {
-            mapATMs.overrideUserInterfaceStyle = .dark
+            mapATMs.overrideUserInterfaceStyle = .light
```

Comment:
How about if we remove the API, instead of just changing it to light?

Reply:
Like @gianpac told me this morning, at this point our wallet forces the `userInterfaceStyle` to be light. I agree that make sense to remove the override. As soon as this `cash-ui` is used by other apps or upstream requires both stylles it will be worth the support of light / dark mode in both platforms.

Reply:
done and pushed